### DISCUSSION TITLE
Assign a hotkey for opening the editor from anywhere

### DISCUSSION THREAD

**Discussion by Kyorie** [ppy/osu#19730] [Ideas]
Use case: A hotkey (_Ctrl+E_?) allows to open the editor with the current map and the current timestamp while in a multiplayer lobby (or basically, anywhere else). It will be helpful for previewing maps in multiplayer.

**Comment by peppy** [comment]
This would cause the game to exit the multiplayer lobby. Would that be your intention?

Regardless, this would never be a hotkey. At most it would be an option from song select. But I kinda disagree with it.

**Reply by Kyorie** [reply]
I guess it'd be fine with a warning about exiting a lobby. I get your point, but i'd like to have it as an additional binding which would be disabled by default.
